git/fetch.go

```diff
@@ -16,6 +16,16 @@ func Fetch(c *Client, opts FetchOptions, rmt Remote, refs []RefSpec) error {
 	opts.FetchPackOptions.All = (refs == nil)
 	opts.FetchPackOptions.Verbose = true
 
+	// If none were provided then we check to see if there are any
+	//  configured refspecs for this remote
+	if refs == nil {
+		// FIXME this only handles one value for the configuration while there can be many with different refspecs for this operation
+		cfg := c.GetConfig(fmt.Sprintf("remote.%s.fetch", rmt))
+		if cfg != "" {
+			refs = []RefSpec{RefSpec(cfg)}
```

Comment:
I think you need to split cfg first in case there's multiple refspecs in the config.

Reply:
Right, that would need to happen before we could get the potential multiple values here.

Reply:
Actually I think I was mistaken about how multiple refspecs are set.. I thought they were space delimited, but according to https://git-scm.com/book/en/v2/Git-Internals-The-Refspec it's done wth multiple fetch config options.